Assets/ScaleMonk Ads/Source/iOS/AdsiOSBinding.cs

```diff
@@ -41,6 +41,11 @@ public void StopBanner(string tag)
             SMAdsStopBanner(tag);
         }
 
+        public void StopBanner()
+        {
+            SMAdsStopBanner("undefined");
```

Comment:
I'd remove the tag parameter from SMAdsStopBanner in the AdsBinding.m file if we don't use it, and call it without the "undefined" string.

Reply:
Done!